src/deeperwin/checkpoints.py

```diff
@@ -68,7 +68,7 @@ def load_data_for_reuse(config: Configuration, raw_config):
     reuse_data = load_run(config.reuse.path)
 
     if config.reuse.reuse_config:
-        config = config.update_configdict_and_validate(config.dict(), build_flattend_dict(raw_config))
+        _, config = config.update_configdict_and_validate(config.dict(), build_flattend_dict(raw_config))
```

Comment:
The return value is now a pair (raw config, parsed config), and here we only need the parsed config